openslides/users/migrations/0002_auto_20150630_0143.py

```diff
@@ -0,0 +1,28 @@
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
```

Comment:
We have a migration here but no model change. Maybe this depends on the Django version so we can omit this change.